### PR TITLE
Added new function 'records.set' for datagrid

### PR DESCRIPTION
**Pull request by trickreich**

### CODE REVIEW

File: dist/husky.js [hunk 1]
```diff
@@ -33484,6 +33484,15 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
             },
 
             /**
+             * listens on and sets one or more data-records with a given ones
+             * @event husky.datagrid.records.set
+             * @param {Object|Array} the new data-record. Must at least contain an id-property. Can also be an array of data-records
```
Comment (danrot) [inline dist/husky.js]: I would only allow an array here

File: dist/husky.js [hunk 2]
```diff
@@ -34676,6 +34686,25 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
             },
 
             /**
+             * Delets all records and sets one or more data-records with a given ones and updates the view
+             * @param records {Object|Array} the new data-record or an array of data-records
```
Comment (danrot) [inline dist/husky.js]: Would also only allow arrays here.

File: dist/husky.js [hunk 3]
```diff
@@ -34676,6 +34686,25 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
             },
 
             /**
+             * Delets all records and sets one or more data-records with a given ones and updates the view
```
Comment (danrot) [inline dist/husky.js]: `Deletes`, and what do you mean with `a given ones`?

**Comment by trickreich** [comment]
@danrot Please have a look once again, thx :)

File: husky_components/datagrid/main.js [hunk 4]
```diff
@@ -1703,6 +1713,21 @@
             },
 
             /**
+             * Deletes all records, sets given records and updateds the view
```
Comment (danrot) [inline husky_components/datagrid/main.js]: `updates`

File: husky_components/datagrid/main.js [hunk 5]
```diff
@@ -1703,6 +1713,21 @@
...
+             * @param records {Array} array of data-records
```
Comment (danrot) [inline husky_components/datagrid/main.js]: Isn't it `@param {Array} records ...`?

**Pull request merged**